assets/src/edit-story/elements/media/edit.js

```diff
@@ -134,8 +141,35 @@ function MediaEdit({ element, box }) {
 
   const isImage = 'image' === type;
   const isVideo = 'video' === type;
+
+  const wheelHandler = useCallback(
+    (evt) => {
+      let newScale = scale || 100;
+      if (evt.deltaY > 0) {
+        newScale += 10;
+      } else {
+        newScale -= 10;
+      }
+      setProperties({ scale: newScale });
+      evt.preventDefault();
+    },
+    [setProperties, scale]
```

Comment:
So, this will be refreshing the callback and by extent rerun `useEffect` on each event including readding event handlers? I think this could be a lot more efficient. E.g. we can inline `wheelHandler` inside the `useEffect` and instead of depending on `scale`, we can simply use `setProperties(function())` that would update scale w/o depending on the current `scale` value.